include/fiction/algorithms/simulation/sidb/is_operational.hpp

```diff
@@ -223,26 +224,42 @@ class is_operational_impl
     [[nodiscard]] sidb_simulation_result<Lyt>
     physical_simulation_of_layout(const bdl_input_iterator<Lyt>& bdl_iterator) noexcept
     {
-        assert(parameters.simulation_parameter.base == 2 && "base number is set to 3");
```

Comment:
Why do you put it in each `ìf`?

Reply:
That was Marcel's suggestion, since the `base != 3` assertion is not needed for `ClusterComplete` specifically.

Reply:
but for QuickExact, QuickSim, and ExGS neither, right?